### PR TITLE
ebos: use the same well model, aquifer model and linear solver as flow

### PR DESCRIPTION
**Pull request by andlaus**
This enables `ebos` to run Norne and other non-trivial data sets. While at it, adapt the tolerances by `ebos`.

This patch only affects the research simulator, i.e. `flow` is unaffected by it.

### CODE REVIEW

**Comment by andlaus** [comment]
jenkins build this please

**Comment by andlaus** [comment]
okay, this PR should be deferred until #1741 is merged...

**Comment by andlaus** [comment]
jenkins build this please

**Comment by andlaus** [comment]
hm, this:
```
(.text._ZN3Opm12StandardWellIN5Ewoms10Properties4TTag11EbosTypeTagEE17assembleControlEqERNS_14DeferredLoggerE[_ZN3Opm12StandardWellIN5Ewoms10Properties4TTag11EbosTypeTagEE17assembleControlEqERNS_14DeferredLoggerE]+0xb74): undefined reference to `Opm::DeferredLogger::warning(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)'
collect2: error: ld returned 1 exit status
```

looks like `libopmsimulators` is not linked against the binaries. I suppose this is either a race condition within jenkins or a problem with the build system. probably the former, so let's try again.

**Comment by andlaus** [comment]
jenkins build this please

**Comment by andlaus** [comment]
hm, weird: this keeps producing undefined symbols for `ebos` in the `flow` well model.

**Comment by andlaus** [comment]
okay, this is probably due to a peculiarity of the default build system (see last patch). let's try again...

**Comment by andlaus** [comment]
jenkins build this please

File: CMakeLists.txt [hunk 1]
```diff
@@ -154,9 +154,12 @@ set_tests_properties(flow__version PROPERTIES
...
-  SOURCES ebos/ebos.cc
+  ONLY_COMPILE
+  ALWAYS_ENABLE
   EXE_NAME ebos
-  ONLY_COMPILE)
+  DEPENDS "opmsimulators"
+  LIBRARIES "opmsimulators"
+  SOURCES ebos/ebos.cc)
 
 if(OPM_GRID_FOUND AND HAVE_ECL_INPUT AND HAVE_ECL_OUTPUT)
```
Comment (akva2) [inline CMakeLists.txt]: preconditions in this repo, nuke the if while at it.

Comment (andlaus) [reply CMakeLists.txt]: fixed. thanks.

**Comment by andlaus** [comment]
jenkins build this please

**Comment by totto82** [comment]
Thanks. Jenkins is happy so I will merge this.

**Pull request merged**